File: backend/src/main/java/com/campus/Campus_Connect/features/connection/dto/ConnectionResponse.java
package com.campus.Campus_Connect.features.connection.dto;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ConnectionResponse {

    private Integer userId;

    private String username;

    private String fullName;

    private String avatarUrl;

    private Integer courseId;

    private Integer admissionYear;

    private ConnectionRelationshipStatus status;
}

File: android-app/app/src/main/java/com/example/campusconnect/feature/profile/data/remote/response/ConnectionResponse.kt
package com.example.campusconnect.feature.profile.data.remote.response

import com.google.gson.annotations.SerializedName

data class ConnectionResponse(
    val userId: Int,
    val username: String,
    val fullName: String,
    val avatarUrl: String?,
    val courseId: Int?,
    val admissionYear: Int?,
    val status: String
)